app/components/Poll/Poll.css

```
@@ -1,16 +1,83 @@
 @import url('~app/styles/variables.css');
 
 .poll {
-  min-height: 160px;
+  background-color: var(--lego-card-color);
+  width: 100%;
+  min-width: 250px;
+  max-width: 500px;
+  min-height: 120px;
+  margin: 0 auto;
+  overflow-y: hidden;
+  border-radius: 5px;
+}
+
+.topBar {
+  width: 100%;
+  z-index: 2;
+}
+
+.pollIcon {
+  display: block;
+  width: 100%;
+  text-align: center;
+  background-color: var(--lego-red-color);
+  color: var(--lego-card-color);
+  padding: 0 0.5rem 0.5rem;
+  box-shadow: 0 2px 2px rgba(0, 0, 0, 20%);
```

```suggestion
  box-shadow: var(--shadow-sm);
```
Not sure which you want

---

Not too sure about this one - all the shadows in variables have either very low opacity or are very spread out, so might have to define some new ones